Tests/Acceptance/AbstractAcceptanceTest.php

```diff
@@ -28,5 +28,8 @@ protected function setUp(): void
         $this->application = new Application($this->client->getKernel());
 
         TestHelper::initializeDoctrineSchema($this->application);
+
+        // https://www.sqlite.org/foreignkeys.html
+        $this->client->getContainer()->get('database_connection')->executeQuery('PRAGMA foreign_keys = ON');
```

Comment:
@HypeMC As this is a sqlite specific query I think we should check first the platform of the connection via the `getDatabasePlatform` method and execute the query only if the platform is actually sqlite. What do you think?

Reply:
@X-Coder264 Done